README.md

```diff
@@ -206,27 +206,27 @@ CDR schema lives in `api/db-cdr` --> all cdr/cohort builder related activities a
 
 There are two commands for deploying the Tanagra app locally along with the Workbench
 
-To deploy the Tanagra api locally:
+To deploy the Tanagra api locally (NOTE: the Tanagra api is dependent on the workbench api being deployed in a different terminal window):
 ```
-./project.rb dev-up-tanagra --disable-auth
+./project.rb dev-up-tanagra
 ```
-The normal `./project.rb dev-up` will still need to be run to deploy the Workbench api
+Optional arguments:
+*  `--disable-auth` Disable authZ
+*  `--version` Deploy a tag (Ex: 0.0.252). Default is :tanagra_tag in environments.rb
+*  `--drop-db` Drop the Tanagra database and recreate
 
 To deploy the Tanagra UI and the Workbench UI together locally:
 ```
 yarn dev-up-tanagra-local
 ```
+Optional arguments:
+*  `-v` Deploy a tag (Ex: 0.0.252). Default is :tanagra_tag in environments.rb
 
 ### Tanagra Deploy to test
 
-To deploy Tanagra api to test (run from `api` directory):
+To deploy Tanagra (api and ui) to test (run from `workbench` directory):
 ```
-./project.rb deploy-tanagra --project all-of-us-workbench-test --version my-version --promote --quiet
-```
-
-To deploy Tanagra UI to test (run from `ui` directory):
-```
-./project.rb deploy-tanagra-ui --project all-of-us-workbench-test --version my-version --promote --quiet
+deploy/project.rb deploy-tanagra --project all-of-us-workbench-test --promote
```

Comment:
When deploying to test we always use :tanagra_tag in environments.rb. This is because we have to generate the tanagra codegen for AoU workbench UI. We are unable to provide a version for the yarn commands, but only environment. We can discuss further offline if you need a better explanation.